VARIABLE.md

```diff
@@ -0,0 +1,128 @@
+# VARIABLES
+
+Variable is used to label and store data in computer memory.
+Variable can update information stored in variable,
+reference or get information stored in a variable.
+
+Its important to distinguish that variable are not value, they contain value and represent them.
+
+`Let` and `const` are new keyword to declare a variable, introduced in `ES6`. Prior to that programmers can declare variable with only the `var` keyword. example:
+
+```js
+var food = 'pizza';
+console.log(food); // output: pizza
+```
+
+`let` keyword signal that values can be reassigned a different value e.g
+
+```js
+let myName = 'Azeez';
+console.log(myName); //output: Azeez
+myName = 'Rotimi';
+console.log(myName); //output: Rotimi
+
+```
+
+If the `let` variable is declared without a value, the variable will be automatically initialized with a value of `undefined` example:
+
+```js
+let myAge;
+console.log(myAge); // output: undefined.
+```
+
+
+The `let` keyword is similar to using the `var` keyword because, anything that can be done with `var` can be done with `let`.
+
+The `const` keyword however cannot because its constant. If you try reassigning a `const` variable, a `TypeError`, will be thrown. example:
```

Comment:
`const` may not mean **constant**. 
Try investigating this also. 
See below for an example.

```js
const array = [1,2,3,4,5]
array                          // [1, 2, 3, 4, 5]
array.push(...[6,7,8,9,10])    // 10
array                          // [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

It is true that we can not reassign a `const` variable, however, it can be mutated and hence may not be a constant in some cases.

Reply:
Understood

Reply:
> According to the link above its a constant.

You are right, buy the point is it can not be reassigned, however, for some data types the value can be mutated (changed). I want you to be aware of this.